src/components/shared/forms/DateRange.vue

```diff
@@ -69,130 +99,203 @@
                 localEnd: moment(this.end),
                 picker: null,
                 stage: 'start',
+                isRange: false,
+                showClear: false,
                 moment,
             }
         },
 
         computed: {
             display() {
-                return `${moment(this.localStart).format('ll')} - ${moment(this.localEnd).format('ll')}`
-            },
-
-            message() {
-                return `Select your ${this.stage} date`
-            },
-
-            computedLocalStart() {
-                return this.localStart.format('YYYY-MM-DD')
+                if (this.isRange) return `${moment(this.localStart).format('ll')} - ${moment(this.localEnd).format('ll')}`
```

Comment:
Are these date formats supposed to be different?

Reply:
Yes if its a single date we have more space for a full date format, if its a range we need to trim it down a bit.

Reply:
Is this ok with Product to have three different date formats? We have had a lot of feedback recently on the consistency of our date formats.

Reply:
The different formats have been cleared with product in this usage case.

Reply:
Ok, I'll talk to product about this, but we can allow it through.